x-pack/plugins/ml/public/embeddables/lens/lens_vis_layer_selection_flyout/flyout.tsx

```diff
@@ -21,14 +25,19 @@ import {
   EuiSpacer,
   EuiText,
 } from '@elastic/eui';
+import { MlApiServices } from '../../../application/services/ml_api_service';
 
-import { FlyoutBody } from './flyout_body';
+import { Layer } from './layer';
 import type { LayerResult } from '../../../application/jobs/new_job/job_from_lens';
 
 interface Props {
   layerResults: LayerResult[];
   embeddable: Embeddable;
   share: SharePluginStart;
+  data: DataPublicPluginStart;
+  application: ApplicationStart;
+  kibanaConfig: IUiSettingsClient;
+  ml: MlApiServices;
```

Comment:
```suggestion
  data: DataPublicPluginStart;
  application: ApplicationStart;
  kibanaConfig: IUiSettingsClient;
  mlApiServices: MlApiServices;
```

Reply:
Updated in https://github.com/elastic/kibana/pull/136421/commits/ae35f0de4bed8502f2c0f97cacee58d0675c6f29